pkg/stub/handler.go

```diff
@@ -155,44 +226,45 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 	case *corev1.Secret:
 		dockercfgSecret, _ := event.Object.(*corev1.Secret)
 
+		if dockercfgSecret.Name != v1alpha1.SamplesRegistryCredentials {
+			return nil
+		}
+
 		if h.samplesResource == nil {
```

Comment:
shouldn't you only be doing this copy/assignment if it's not a delete event?

Reply:
will add that conditional